### DISCUSSION TITLE
Getting status code 500 with jenkins.

### DISCUSSION THREAD

**Discussion by thakuravneet** [NetzOptimize/pyQA#15] [Q&A] [answered]
I have deployed the framework on Jenkins and some how getting this error please help 

![image](https://user-images.githubusercontent.com/108736128/192230986-c25fd11a-693b-432f-843f-ec175ac87011.png)

**Comment by Montekkundan** [comment, marked as answer]
Hello, could you please tell what version of Jenkins you're using.
I believe there are few things you can try to clear this issue:
- Try restarting Jenkins
- Your CPU or RAM usage is being used over its limit, and you might need to cut a little down on your test cases or increase the system power.

**Reply by thakuravneet** [reply]
The problem is fixed, it was the CPU and RAM being used over its limit.
